lib/apollo/hooks/actions.js

```diff
@@ -54,6 +57,8 @@ export const useSignUp = () => {
     try {
       await mutation({ variables: { email, password, ...rest } });
 
+      window.localStorage.setItem(SIGN_IN_EVENT, Date.now());
```

Comment:
Should we add some tests for this at next PR?

Reply:
I've created an issue for that https://github.com/fs/react-next-base/issues/83